Libplanet.Tests/Libplanet.Tests.csproj

```diff
@@ -33,6 +33,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AsyncEnumerator" Version="4.0.2" />
```

Comment:
Why is it needed? `AsyncEnumerator` seemed to be replaced with `System.Linq.Async` on [the line](https://github.com/planetarium/libplanet/pull/798/files#diff-3bf6e3fb30349a8654faf74cc804ad61R96)

Reply:
This is only for tests:

> Although this removed the dependency on *AsyncEnumerator* package and introduce the new syntax into *Libplanet* project, note *Libplanet.Tests* project still uses C# 7.2 and the third-party package, which is intended.